livekit_ingress.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package livekit;
+
+import "google/protobuf/empty.proto";
+import "livekit_models.proto";
+
+option go_package = "github.com/livekit/protocol/livekit";
+
+service Ingress {
+    rpc CreateIngress(CreateIngressRequest) returns (IngressInfo);
+    rpc ListIngress(ListIngressRequest) returns (ListIngressResponse);
+    rpc DeleteIngress(DeleteIngressRequest) returns (google.protobuf.Empty);
```

Comment:
Should this have a response of success/failure at least?

Reply:
failures would be signaled with a different status code in Twirp-land. (i.e. 500)